test/controllers/testOnly/FeatureSwitchControllerSpec.scala

```diff
@@ -17,14 +17,31 @@
 package controllers.testOnly
 
 import base.SpecBase
-import config.featureSwitches.{CallAPI1812ETMP, FeatureSwitching}
+import config.AppConfig
+import config.featureSwitches.{CallAPI1812ETMP, FeatureSwitch, FeatureSwitching}
+import org.mockito.Mockito.{mock, reset, when}
+import org.mockito.Matchers.any
+import play.api.Configuration
 import play.api.test.FakeRequest
 import play.api.test.Helpers._
+import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
+
+import java.time.LocalDate
 
 class FeatureSwitchControllerSpec extends SpecBase with FeatureSwitching {
   val controller: FeatureSwitchController = injector.instanceOf[FeatureSwitchController]
+  val mockConfig = mock(classOf[Configuration])
+  val mockServicesConfig = mock(classOf[ServicesConfig])
+  val config: AppConfig = new AppConfig(mockConfig, mockServicesConfig)
+
+  class Setup {
+    reset(mockConfig)
+    reset(mockServicesConfig)
+    val controller = new FeatureSwitchController(mcc)(config)
+    FeatureSwitch.listOfAllFeatureSwitches.foreach(sys.props -= _.name)
```

Comment:
Could you add 
```suggestion
   sys.props -= featureSwitching.TIME_MACHINE_NOW
```
so that the time machine date is removed for the next spec